eml_to_pdf.gemspec

```diff
@@ -4,25 +4,25 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'eml_to_pdf/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "eml_to_pdf"
+  spec.name          = 'eml_to_pdf'
   spec.version       = EmlToPdf::VERSION
-  spec.authors       = ["Yves Siegrist"]
-  spec.email         = ["Elektron1c97@gmail.com"]
+  spec.authors       = ['Yves Siegrist', 'Hubert Pomorski']
+  spec.email         = %w(Elektron1c97@gmail.com hubert.pomorski@sabre.com)
```

Comment:
I respect that you want to be in this list, but I'm still the author of this gem, if you're planning to maintain it with me together, this could be changed, but not only for one PR :grin: